File: Iniciante/1143.js
let lim = lines.shift();
let n1 = 1;
let n2 = 1;
let n3 = 1;

for (let i = 0; i < lim; i++) {
  console.log(n1 + ' ' + n2 + ' ' + n3);
  n1++;
  n2 = n1**2;
  n3 = n1*n2;
}

File: Iniciante/1143.py
lim = int(input())
n1 = 1
n2 = 1
n3 = 1

for i in range(lim):
  print("{} {} {}".format(n1, n2, n3))
  n1 += 1
  n2 = n1**2
  n3 = n1*n2
